staging/src/k8s.io/api/autoscaling/OWNERS

```diff
@@ -10,4 +10,3 @@ reviewers:
   - sttts
   - ncdc
   - dims
-  - errordeveloper
```

Comment:
Should this go to `emeritus_reviewers`?  On casual reading I expected it to.

Reply:
Or did we just decide no `emeritus_reviewers` in the files?

Reply:
@smarterclayton we don't have a `emeritus_reviewers` and the consensus was to just drop inactive reviewers. please see https://github.com/kubernetes/community/pull/3627#issuecomment-487253671